### ISSUE TITLE
Uninstall more than Facebook?

### ISSUE THREAD

**Issue by darthmall** [Andy-set-studio/npm-uninstall-facebook#15]
Could the focus of this resource be expanded beyond just Facebook and include as many surveillance-based companies as necessary? Google seems to [behave just as badly as Facebook](https://techcrunch.com/2019/01/30/googles-also-peddling-a-data-collector-through-apples-back-door/), for example, which would rule out Angular as an alternative to React.

**Comment by andybelldesign** [comment]
Strong agree. I've already rejected #8 for that reason.

The only problem is the domain though...

**Comment by darthmall** [comment]
Yeah, that's a good point. "npm uninstall facebook" is a good name; "npm uninstall surveillance-capitalists" doesn't really roll off of the tongue. 😅 I kind of like "npm uninstall big-brother", although that's a reference that might be a little opaque, and may also prove to be limiting: conceivably a company might engage in other ethically questionable behaviors that aren't necessarily surveillance-related.

**Comment by andybelldesign** [comment]
I tell you what, there would be value in a sister-site, npm-uninstall-google.com

**Comment by IngwiePhoenix** [comment]
Just some random thoughts: 
- `npm-uninstall-monopoly`
- `npm-uninstall-bloat`
- `npm-uninstall-corp`

I like this project a lot already! :)

**Comment by cedeber** [comment]
`npm-uninstall-bullshit` 😅

**Comment by kenokenobingo** [comment]
`npm-install-freedom`

![RMS](https://i.imgur.com/Sec6Yei.jpg)
